.github/workflows/conformance.yml

```diff
@@ -1,49 +1,40 @@
 name: Conformance Testing
 
 on:
-  push:
-    branches:
-      - main
-      - release-*
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+*"
-  pull_request:
-  schedule:
-    - cron: "0 4 * * *" # run every day at 4am UTC
+  workflow_call:
+    inputs:
+      image:
+        required: true
+        type: string
+      k8s-version:
+        required: true
+        type: string
+      enable-experimental:
+        required: true
+        type: boolean
 
 defaults:
   run:
     shell: bash
 
-concurrency:
-  group: ${{ github.ref_name }}-conformance
```

Comment:
Does this now fall under the parent caller? Does it not make sense to have this logic anymore?

Reply:
TIL that you can have concurrency even at the job level, so I guess it doesn't hurt to leave it in this workflow.

Reply:
well actually I just realized why I removed it in the first place...the workflow now gets called from CI in parallel so if we want to run them all at the same time, we need to remove the concurrency group